test/e2e/multicontainer_test.go

```diff
@@ -0,0 +1,104 @@
+// +build e2e
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	pkgTest "knative.dev/pkg/test"
+	"knative.dev/pkg/test/logstream"
+	"knative.dev/serving/pkg/apis/config"
+	"knative.dev/serving/test"
+	v1test "knative.dev/serving/test/v1"
+)
+
+func TestMultiContainer(t *testing.T) {
+	t.Parallel()
+	cancel := logstream.Start(t)
+	defer cancel()
+
+	clients := Setup(t)
+
+	featureConfigMap, err := rawCM(clients, config.FeaturesConfigName)
+	if err != nil {
+		t.Errorf("Error retrieving config-feature configmap: %v", err)
+	}
+
+	patchedFeatureConfigMap := featureConfigMap.DeepCopy()
+	patchedFeatureConfigMap.Data["multi-container"] = "enabled"
+	_, err = patchCM(clients, patchedFeatureConfigMap)
+	defer patchCM(clients, featureConfigMap)
+	test.CleanupOnInterrupt(func() { patchCM(clients, featureConfigMap) })
```

Comment:
We don't do these in tests anymore. We either need to run this as a separate test-suite and enable the feature via script before running it, or (which would be my preference) we just enable it in tests anyway by adding a configmap with the feature enabled to `/test/config`

Reply:
Yep i have updated similar way like `enable_tag_header_based_routing`